content/docs/authorization/troubleshooting.md

```diff
@@ -58,9 +58,9 @@ View the contents /tmp/policy-install-for-karavi* file listed in the error messa
 ```
 $ cat /tmp/policy-install-for-karavi3163047435
 
-# find the location of the policy-install.sh script located in the file and manually run the script
+# find the location of the policy-install.sh script located in the extracted folder of the tar archive and manually run the script
```

Comment:
This issue will not occur anymore since the policy-install.sh script now runs outside the rpm. The resolution now is to use the fixed version v1.5.0

Reply:
Done please review